src/rdb/rdb_raft.c

```diff
@@ -1987,13 +1999,19 @@ rdb_timerd(void *arg)
 		if (rc != 0)
 			D_ERROR(DF_DB": raft_periodic() failed: %d\n",
 				DP_DB(db), rc);
+		if (db->d_stop)
+			break;
 
-		t_prev = t;
 		/* Wait for d in [d_min, d_max] before the next beat. */
 		d = d_min + (d_max - d_min) * rdb_raft_rand();
-		while ((t = ABT_get_wtime()) < t_prev + d && !db->d_stop)
-			ABT_thread_yield();
+		sched_req_sleep(sched_req, (uint32_t)(d * 1000));
```

Comment:
I think it's likely not an issue: The original code "sleeps" until a deadline t_prev + d, while the new code sleeps for duration d = t_prev + latency_of_raft_periodic + d. The extra latency should be ignorable.

Reply:
it could be a problem, let me try to change it back to original behavior.